src/example4.cpp

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: Intel Corporation
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <dr/mhp.hpp>
+#include <fmt/core.h>
+
+namespace mhp = dr::mhp;
+using T = int;
+
+int main(int argc, char **argv) {
+
+  mhp::init(sycl::default_selector_v);
+  std::size_t xdim = 9, ydim = 5;
+
+  std::array<std::size_t, 2> extents2d = {xdim, ydim};
+
+  mhp::distributed_mdarray<T, 2> a(extents2d);
+  mhp::distributed_mdarray<T, 2> b(extents2d);
+  mhp::distributed_mdarray<T, 2> c(extents2d);
```

Comment:
Please add a comment to lines 21-22, encouraging a tutorial user to change the initial content of arrays a & b.

Reply:
Done